### PR TITLE
Adjust text for OKTA-567669 (CSP)

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Wording change to CSP item to match changes from prod.
- **Is this PR related to a Monolith release?** 2023.01.2

### Resolves:

* [OKTA-568344](https://oktainc.atlassian.net/browse/OKTA-568344)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ A 30-second clock skew is now allowed for access and ID tokens to validate that
 
 #### Content Security Policy update
 
-To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
+Over the next few months we are gradually releasing enhancements to our [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers. During this time you may notice an increase in header sizes. <!--OKTA-567669 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:24]:
```suggestion
Over the next few months, we are gradually releasing enhancements to our [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers. During this time you may notice an increase in header sizes. <!--OKTA-567669 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ A 30-second clock skew is now allowed for access and ID tokens to validate that
 
 #### Content Security Policy update
 
-To improve our security and protect against Cross-Site Scripting (XSS) attacks, we are expanding the use of [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to include when requests come from custom domains on non-customizable pages. <!--OKTA-567669 -->
+Over the next few months we are gradually releasing enhancements to our [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers. During this time you may notice an increase in header sizes. <!--OKTA-567669 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:21]:
```suggestion
Over the next few months, we are gradually releasing enhancements to our [Content Security Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers. During this time you may notice an increase in header sizes. <!--OKTA-567669 -->
```

**Review by susanharper-okta — APPROVED** [review]
one nit - the lack of comma isn't exactly incorrect...do with that what you want. approving to not hold you up.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 1 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:24]: I'll go with it.

**Pull request merged**